### PR TITLE
FIX: Convert Source File Path to Local Path in Error

### PR DESCRIPTION
**Pull request by nc-x**

### CODE REVIEW

File: compiler.r [hunk 1]
```diff
@@ -138,7 +138,7 @@ red: context [
 			either word? err [
 				join uppercase/part mold err 1 " error"
 			][reform err]
-			"^/*** in file:" mold script-name
+			"^/*** in file:" mold to-local-file script-name
```
Comment (dockimbel) [inline compiler.r]: `mold to-local-file` will add unwanted double quotes, so `mold` should be removed in such case.

Comment (nc-x) [inline compiler.r]: @dockimbel done

**Comment by nc-x** [comment]
@dockimbel Done, and I merged the two commits so as to not pollute the pull request (though I think the commits could have been squashed when you merged the PR, but I was not sure if it was possible, so I did it myself). But the appveyor and travis builds seem to be stuck and need to be manually stopped (I think).

**Comment by dockimbel** [comment]
`when you merged the PR` I haven't merged it.

**Comment by nc-x** [comment]
`when you merged the PR`
 *merge 😉 😄

**Pull request merged**